sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/FaultInjectionBridgeInternal.java

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import com.azure.cosmos.models.FaultInjectionRule;
+import reactor.core.publisher.Mono;
+
+import java.util.List;
+
+
```

Comment:
Should also add @Warning with statement along the lines of class + methods

"Don't us in production - this is to allow failure injection in test environments only."

Reply:
My preference would also be to use real verb - like configureFaultINjectionRules - I think that is what we do for metrics as well.

Reply:
Also worth thinking whether we should rename it (not have BridgeInternal suffix - like CosmosFaultInjection or CosmosFaultInjectionHelper - I would also mark these APIs a s @Beta for now

We are not supposed to add new bridge APIs - and given that we want cx to use these APIs I would also not start with it. But also feel like this API should not be on Container itself - to allow adding the warning etc. and clarify that this is not meant to be used in prod. So maybe above is a reaonoable compromise?

Reply:
yea good point, added @warning on CosmosFaultInjectionHelper class
Added @beta on all newly added models